packages/server/lib/controllers/connect/postSessions.ts

```diff
@@ -0,0 +1,121 @@
+import type { PostConnectSessions } from '@nangohq/types';
+import { z } from 'zod';
+import db from '@nangohq/database';
+import { asyncWrapper } from '../../utils/asyncWrapper.js';
+import { validateRequest } from '@nangohq/utils';
+import * as keystore from '@nangohq/keystore';
+import * as linkedProfileService from '../../services/linkedProfile.service.js';
+import * as connectSessionService from '../../services/connectSession.service.js';
+
+const validate = validateRequest<PostConnectSessions>({
```

Comment:
I know the format is frustrating rn but I wouldn't use this, the error codes are entirely different from the rest of the API

Reply:
changed it but it is a bit ridiculous to have to repeat the same 10 lines on every single endpoint

Reply:
yes, very happy to iterate on something similar to validateRequest, just need to follow what we previously set to avoid breaking change. It's mostly a matter of copying the code and changing the response